### DISCUSSION TITLE
How it is different from glommio ?

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [bytedance/monoio#26] [General]
Hi guys.
What is the difference between your framework and glommio?  
https://github.com/DataDog/glommio

**Comment by dyxushuai** [comment]
The most important differences are the IO api design and the performance-latency tradeoff.
For the latter you can see our benchmark doc.
Anyway the details between them are a long story. Maybe we will write an article to introduce it properly in the near future.🥰

**Reply by andrii0lomakin** [reply]
I see, but glommio provides better support for file system operations. Do you have plans to extend it? I have got the impression that monio is concentrated mostly on the implementation of network operations. Is my impression wrong?

**Reply by dyxushuai** [reply]
Yes, you are right. But support file IO is our import milestone.

**Reply by audacioustux** [reply]
@dyxushuai hi, is there any article written already by this time? was looking for the design choices / differences between the two runtime...
